src/main/java/me/dev/oliver/youtubesns/service/UserLoginServiceImpl.java

```diff
@@ -30,7 +30,7 @@ public boolean signin(UserDto user) {
     boolean result = userService.findByIdAndPw(user);
 
     if (result) {
-      SessionUtil.getHttpSession().setAttribute(SessionKeys.USER_ID, user.getUserId());
+      LoginSessionUtil.getHttpSession().setAttribute(SessionKeys.USER_ID, user.getUserId());
```

Comment:
이곳은 **세션에서 특정한 값을 가져온다**라는 로직을 캡슐화시켜주면 좋을 것 같네요. 그냥 일반적인 방식으로 세션을 불러와도 되는데 굳이 유틸클래스를 통해서 세션을 가져와야할 필요는 없어보여서요~

```suggestion
      SessionUtils.set(SessionKeys.USER_ID, user.getUserId());
```